_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -22,28 +22,28 @@ acknowledgments:
 
 ## Applicability
 
-Any [WAI-ARIA 1.1 state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is not empty (""), and that is specified on an HTML or SVG element.
+The rule applies to any HTML or SVG element that has any not empty (`""`) [WAI-ARIA 1.1 state or property][] specified.
 
 ## Expectation
 
-Each test target has a valid value according to its [WAI-ARIA 1.1 value type](https://www.w3.org/TR/wai-aria-1.1/#propcharacteristic_value).
+Each test target has a valid value according to its [WAI-ARIA 1.1 value type][].
```

Comment:
That is not correct anymore. With the new Applicability, the test targets are elements, not `aria-*` attributes. HTML/SVG elements do not have a "WAI-ARIA value type".

Reply:
This should solve this.